src/vs/workbench/contrib/terminal/browser/links/terminalLocalLinkDetector.ts

```diff
@@ -67,11 +67,11 @@ export const lineAndColumnClause = [
 
 const fallbackMatchers: RegExp[] = [
 	// Python style error: File "<path>", line <line>
-	/^\s*File (?<link>"(?<path>.+)"(, line (?<line>\d+))?)/,
+	/^ *File (?<link>"(?<path>.+)"(, line (?<line>\d+))?)/,
```

Comment:
Do we also need a trailing ` *` for this regexp?

Reply:
Trailing space generally shouldn't show up as we trim it, I also notice `$` isn't used so it's actually not needed for the other one